src/mobu/exceptions.py

```diff
@@ -542,6 +885,72 @@ def to_slack(self) -> SlackMessage:
 
         return message
 
+    def common_fields(self) -> list[SlackBaseField]:
+        """Return common fields to put in any alert.
+
+        Returns
+        -------
+        list of SlackBaseField
+            Common fields to add to the Slack message.
+        """
+        fields = super().common_fields()
+        fields.extend(self.mobu_fields())
+        return fields
+
+    @classmethod
+    def from_client_exception(
+        cls,
+        exc: ClientJupyterWebSocketError,
+        monkey: str | None = None,
+        event: str | None = None,
+        annotations: dict[str, str] | None = None,
+        started_at: datetime.datetime | None = None,
+    ) -> Self:
+        """
+        Add Mobu-specific fields to exception from NubladoClient layer.
+
+        Parameters
+        ----------
+        exc
+            Original exception
+        monkey
+            Monkey spawning the lab, if known.
+        event
+            Event (from mobu's perspective) spawning the lab, if known.
+        annotations
+            Additional annotations
+        started_at
+            Timestamp for beginning of operation that caused the exception,
+            if known.
+
+        Returns
+        -------
+        JupyterWebSocketError
+            Converted exception.
+        """
+        anno: dict[str, str] = {}
+        if annotations is not None:
+            anno = annotations
+        body = exc.body
+        if body is not None:
+            body_bytes = body.encode()
```

Comment:
Is it worth storing the encoded body on the nublado `JupyterWebSocketException`?

Reply:
Imma say "not at this point", although when we do the big Slack Exception refactor, the answer there is "yes" or maybe even better, "we'll get it for free because that's how the restructure will work".